docs/user-guide/analysis.md

```diff
@@ -59,15 +63,167 @@ be found
 
 #### Example: Reserving a single node for 20 minutes for interactive analysis
 
+    ```bash
     auser@uan01:> salloc --nodes=1 --tasks-per-node=1 --cpus-per-task=1 \
                   --time=00:20:00 --partition=standard --qos=short \
                   --reservation=shortqos --account=[budget code]
-
+    ```
 
 ## Data analysis nodes
 
-The full ARCHER2 system will have dedicated data analysis nodes. Further
-information on these nodes will be added here once it becomes available.
+The data analysis nodes on the ARCHER2 full system are designed for large
+compilations, post-calculation analysis and data manipulation. They should 
+be used for jobs which are too small to require a whole compute node, but 
+which would have an adverse impact on the operation of the login nodes if 
+they were run interactively.
+
+Unlike compute nodes, the data analysis nodes are able to access the `/home`, 
+`/work`, and `/rdfaas` directories. They can also be used to transfer data 
+from a remote system to ARCHER2 and vice versa (using e.g. `scp` or `rsync`).
+This can be useful when transferring large amounts of data that might take 
+hours to complete.
+
+The ARCHER2 data analysis nodes can be reached by using the `serial` 
+partition and the `serial` queue. Unlike other nodes on ARCHER2, you may 
+only request part of a single node and you will likely be sharing the node 
+with other users.
+
+The data analysis nodes are set up such that you can specify the number of 
+cores you want to use (up to 32 physical cores) and the amount of memory you 
+want for your job (up to 128 GB). You can have multiple jobs running on the 
+data analysis nodes at the same time, but the total number of cores used by 
+jobs currently running can not exceed 32 cores, and the total memory used 
+by jobs currently running cannot exceed 128 GB -- any jobs above this limit 
+will remain pending until your previous jobs are finished.
+
+!!! note:
+    When running on the data analysis nodes, you must always specify either 
+    the number of cores you want, the amount of memory you want, or both. 
+    The examples shown below specify the number of cores with the `--ntasks`
+    flag and the memory with the `--mem` flag. If you are only wanting to 
+    specify one of the two, please remember to delete the other one.
+
+#### Example: Running an MPI batch script on the data analysis nodes
```

Comment:
While it is potentially useful to show how to run MPI executables on the data analysis nodes I think our examples should really show how to run serial data analysis scripts (I know you did this above) . I think we should replace the mpi example with a serial example here as that is the use mode we should be promoting most heavily